scopes/envs/envs/envs.cmd.tsx

```diff
@@ -12,7 +12,7 @@ import { EnvOverview } from './components/env-overview';
 
 export class ListEnvsCmd implements Command {
   name = 'list';
-  description = 'list all envs available in the workspace';
+  description = 'list all envs available in the workspace'; // @david can we provide a bit more detail on what defines an 'available' env
```

Comment:
There are the envs that either exist in the workspace (.bitmap) or envs of existing workspace components. (because when you load a component, even if its env is not in .bitmap, we load it, so the env is available)

Reply:
what are the envs from the workspace.json root? You mean those added with their component-id in the workspace.jsonc?
And do envs that are currently used for components, though not locally registered, also make it to this list?

underlying all these questions - I'm struggling to work out what this is used for?

Reply:
I'm not either what this is used for.  
But yes, I mean these components that you were running "bit use" for and they're in the root of the workspace.jsonc.

The envs of the components - it depends. They get registered once you load the components. But when you run the command, you don't load all components, so technically they won't be there. 

I agree, we probably need to change this command.. not sure what/how. Probably Gilad/Itay/Ran can help here.

Reply:
spoke to itay - will change it to `envs used in the workspace` to make it clearer what it outputs